packages/cli/src/tests/synthql.config.ts

```diff
@@ -0,0 +1,26 @@
+import { CliConfig } from '../types/CliConfig';
+
+export const config: CliConfig = {
+    out: 'src/generated/synthql',
+    defaultSchema: 'public',
+    schemas: ['public'],
+    tables: ['actor', 'customer'],
+    schemaDefOverrides: {
```

Comment:
did you check if the overrides will work when you have two tables with the same name defined in different schemas?

Reply:
To fix I suggest you require schema.table notation (e.g. public.actor). Its slightly more verbose, but more explicit.